server/models/migrations/next/migrate.sql

```diff
@@ -1,9 +1,23 @@
 -- change the pavillion table to ward table
-RENAME TABLE pavillion TO ward;
+-- RENAME TABLE pavillion TO ward;
```

Comment:
It looks like you decided to create a new table for ward.  Can we change this to `drop table pavillion` then?

Reply:
I made this update, just because the table pavillon had already been renamed by ward even in the Tshikaji database and Branch Master